### PR TITLE
Search filtering functionality

### LINKED ISSUE #34: Implement Results view

### PR DESCRIPTION
**Pull request by Karyum**
close #34 #37 
:maple_leaf:

### CODE REVIEW

File: src/routes/results.js [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
-  res.render('search_results');
+  let options = {};
+  const filtersArray = [
+    req.query.search ? 'search-' : 'null-',
+    req.query.stage ? 'stage-' : 'null-',
+    req.query.sector ? 'sector' : 'null'
+  ].join('')
+
+  switch (filtersArray) {
+    case 'search-stage-sector':
```
Comment (Karyum) [inline src/routes/results.js]: Hey @Suuuuha esraa is saying that the case statements are not readable or not understandable any idea how to change it ?? maybe change the null to 'noFilter'?

Comment (Karyum) [inline src/routes/results.js]: also im going to be adding tests so dont approve just yet

Comment (Suuuuha) [inline src/routes/results.js]: maybe change the null to 'null-the filter missing' for example in your second case 'search-stage-noSector'

**Comment by Suuuuha** [comment]
@Karyum fix your conflict and I'll approve it :smiling_imp:

**Comment by Karyum** [comment]
@Suuuuha done :maple_leaf:

**Comment by esraajb** [comment]
@Karyum merge conflict

**Comment by Karyum** [comment]
@esraajb resolved

**Pull request merged**